mapentity/static/mapentity/leaflet-datatables.js

```diff
@@ -103,6 +103,26 @@ L.MapListSync = L.Class.extend({
     },
 
     _reloadList: function (refreshLayer) {
+        var formData = new FormData(document.querySelector('#mainfilter'));
+        var filter = false;
+
+        for (var value of Array.from(formData)) {
+            if (value[0] !== 'bbox') {
+                if (value[1] !== '') {
+                    filter = true;
+                }
+            }
+        }
+        if (filter) {
+            $('#filters-btn').removeClass('btn-info');
+            $('#filters-btn').addClass('btn-warning');
+        }
+        else {
+            $('#filters-btn').removeClass('btn-warning');
+            $('#filters-btn').addClass('btn-info');
+        }
```

Comment:
You can simplify and use toggleClass jQuery method by using filter boolean as second parameter
```
$('#filters-btn').toggleClass('btn-info', !filter);
$('#filters-btn').toggleClass('btn-warning', filter);
```